app/models/search/work_search_form.rb

```diff
@@ -126,8 +126,15 @@ def summary
     unless tags.empty?
       summary << "Tags: #{tags.uniq.join(", ")}"
     end
-    if %w(1 true).include?(self.complete.to_s)
```

Comment:
For the new options, just checking for T/F is fine, but "Complete" is there in old search so we still need to check for `1 true`.

You can add a comment:

```
# ES UPGRADE TRANSITION #
# Remove checks for 1 and true
```
so we can deal with it later.

Reply:
Just to confirm, old search would still use `models/search/work_search_form.rb`? I had thought that old search stuff picked `models/work_search.rb` as the model. :thinking:

Reply:
Oh you're right! This is good, I got it backwards. ^^

Reply:
Haha, all good. Keep me honest :coffee: